multiclusterhub/charts/insights-chart/templates/policyreport-servicemonitor.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2021 Red Hat, Inc.
+# Copyright Contributors to the Open Cluster Management project
+
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: acm-{{ .Values.insights.name }}
+  namespace: openshift-monitoring
+  labels:
+    app: {{ .Values.insights.name }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    component: {{ .Values.insights.name }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/name: {{ .Values.insights.name }}
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+spec:
+  endpoints:
+  - interval: 60s
+    port: https
+    scheme: https
+    scrapeTimeout: 10s
+    bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    tlsConfig:
+      insecureSkipVerify: true
+  jobLabel: insights-app
+  selector:
+    matchLabels:
+      insights-app: {{ .Values.insights.name }}
+  namespaceSelector:
+    matchNames:
+      - {{ .Release.Namespace }}
```

Comment:
Missing newline at end of last line.

https://stackoverflow.com/a/729795/1437822